plugins/chat/spec/system/admin/csv_exports_spec.rb

```diff
@@ -12,6 +12,8 @@
   end
 
   it "exports chat messages" do
+    Capybara.save_path = Downloads::FOLDER
```

Comment:
The playwright drive is using this save_path deep in the gem when saving something, it was the easiest way to go around this.